snforge_std/src/cheatcodes.cairo

```diff
@@ -20,6 +20,36 @@ pub enum CheatSpan {
     TargetCalls: usize,
 }
 
+/// Enum used to specify the call data that should be matched when mocking a contract call.
+#[derive(Copy, Drop, PartialEq, Clone, Debug)]
+pub enum MockCallData {
+    /// Matches any call data.
+    Any,
+    /// Matches the specified serialized call data.
+    Values: Span<felt252>,
+}
+
+impl MockCallDataSerde of Serde<MockCallData> {
+    fn deserialize(ref serialized: Span<felt252>) -> Option<MockCallData> {
+        let value: Option<Option<Span<felt252>>> = Serde::deserialize(ref serialized);
+
+        match value {
+            Option::None => Option::None,
+            Option::Some(call_data) => match call_data {
+                Option::None => Option::Some(MockCallData::Any),
+                Option::Some(data) => Option::Some(MockCallData::Values(data)),
+            },
+        }
+    }
+
+    fn serialize(self: @MockCallData, ref output: Array<felt252>) {
+        match self {
+            MockCallData::Any => Option::<Span<felt252>>::None.serialize(ref output),
+            MockCallData::Values(data) => Option::Some(*data).serialize(ref output),
+        }
+    }
+}
+
```

Comment:
Why not just derive it?

```suggestion
/// Enum used to specify the call data that should be matched when mocking a contract call.
#[derive(Copy, Drop, PartialEq, Clone, Debug, Serde)]
pub enum MockCallData {
    /// Matches any call data.
    Any,
    /// Matches the specified serialized call data.
    Values: Span<felt252>,
}
```

Reply:
Source code updated in cheatnet crate introduce `MockCalldata` enum